src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +544,26 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
+
+			var path = localPath;
+			var pos = path.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1) path = path.Substring(pos + 1);
+
+			// try to find the css file
+			path = CurrentBook.GetFileLocator().LocateFile(path);
```

Comment:
BloomFileLocator will certainly look for things in the book folder and the collection folder. In fact it will look for everything there as  a last resort! However, the old code had specific preferences for finding languageDisplay.css and CollectionStyles.css in these places only. I don't know whether this change matters (or maybe even is intentional) but you might want to check with JohnH.